src/sconelib/scone/measures/MimicMeasure.cpp

```diff
@@ -22,8 +22,8 @@ namespace scone
 		INIT_MEMBER( pn, include_states, xo::pattern_matcher( "*" ) ),
 		INIT_MEMBER( pn, exclude_states, xo::pattern_matcher( "" ) ),
 		INIT_MEMBER( pn, use_best_match, false ),
-		INIT_MEMBER( pn, average_error_limit, 1 ),
```

Comment:
Sorry, it included the update to the MimicMasure. Do you want me to update this?